optimum/intel/openvino/quantization.py

```diff
@@ -165,14 +165,29 @@ def __init__(
         self.apply_caching = apply_caching
         self.inference_result_mock = inference_result_mock
         self.tensor_cache = {}
+        self.stateful = len(request.query_state()) > 0
+        self._reset_state_called = False
 
     def collect_inputs(self, inputs):
+        if self.stateful:
+            if isinstance(inputs, dict) and is_nncf_version(">", "2.19"):
```

Comment:
this would include any following patch release (if any) shouldn't we have instead ?
```suggestion
            if isinstance(inputs, dict) and is_nncf_version(">=", "2.20"):
```

Reply:
Good point, updated